src/components/table/components/table-row/table-row.tsx

```diff
@@ -23,7 +23,7 @@ const RowLayout = ({ children, layoutType, ...props }: any) => {
 const DefaultGroupHead = ({ row, layoutType, style }: any) => {
   const rowProps = row.getRowProps()
   return layoutType === "table" ? (
-    <tr {...rowProps} style={style}>
+    <tr title="group-head" {...rowProps} style={style}>
```

Comment:
Is it some a11y/semantic requirement?

Reply:
actually it's just for testing purposes, but yes accessibility wise it's better than not having anything :)